src/DiffEqGPU.jl

```diff
@@ -1,58 +1,41 @@
 module DiffEqGPU
 
-using GPUifyLoops, CuArrays, CUDAnative, DiffEqBase, LinearAlgebra, Distributed
+using KernelAbstractions, CuArrays, CUDAnative, DiffEqBase, LinearAlgebra, Distributed
```

Comment:
This might need to make use of CUDA.jl? @vchuravy does KA require using CUDA.CuArrays and such? If so, we might need to update this all at once?

Reply:
The newer versions of KA use CUDA.jl